Schema/pullRequest.schema.json

```diff
@@ -6,45 +6,33 @@
     "type": "object",
     "properties": {
         "merged_at": {
-            "type": [
-                "string",
-                "null"
-            ],
-            "format": "date-time"
+            "type": "string",
+            "format": "date-time",
+            "nullable": true
         },
         "diff_url": {
-            "type": [
-                "string",
-                "null"
-            ],
-            "format": "uri"
+            "type": "string",
+            "format": "url",
+            "nullable": true,
+            "required": true
         },
         "html_url": {
-            "type": [
-                "string",
-                "null"
-            ],
-            "format": "uri"
+            "type": "string",
+            "format": "url",
+            "nullable": true,
+            "required": true
         },
         "patch_url": {
-            "type": [
-                "string",
-                "null"
-            ],
-            "format": "uri"
+            "type": "string",
+            "format": "url",
+            "nullable": true,
+            "required": true
         },
         "url": {
-            "type": [
-                "string",
-                "null"
-            ],
-            "format": "uri"
+            "type": "string",
+            "format": "url",
+            "nullable": true,
+            "required": true
         }
-    },
-    "required": [
-        "diff_url",
-        "html_url",
-        "patch_url",
-        "url"
-    ]
+    }
```

Comment:
Yeah, I think the best apporach here was to ask Ruben, as it's a design choice. For maintainability *and* overview, keeping the required array is a better approach. Also, if you look at the other Bundles, the array approach is standard.